config/locales/status_tags/delivery_partner_participant_status.yml

```diff
@@ -27,168 +27,168 @@ en:
 
       # mentoring
 
       active_mentoring:
         id: "training_or_eligible_for_training"
         label: "Training or eligible for training"
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        description: "We’ve confirmed this participant is eligible for this programme. Your training provider will contact them directly."
 
       not_yet_mentoring:
         id: "training_or_eligible_for_training"
         label: "Training or eligible for training"
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        description: "We’ve confirmed this participant is eligible for this programme. Your training provider will contact them directly."
 
       active_mentoring_ero:
         id: "training_or_eligible_for_training"
         label: "Training or eligible for training"
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        description: "We’ve confirmed this participant is eligible for this programme. Your training provider will contact them directly."
 
       not_yet_mentoring_ero:
         id: "training_or_eligible_for_training"
         label: "Training or eligible for training"
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        description: "We’ve confirmed this participant is eligible for this programme. Your training provider will contact them directly."
 
       # changes of circumstance
 
       leaving:
         id: "training_or_eligible_for_training"
         label: "Training or eligible for training"
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        description: "We’ve confirmed this participant is eligible for this programme. Your training provider will contact them directly."
 
       joining:
         id: "training_or_eligible_for_training"
         label: "Training or eligible for training"
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        description: "We’ve confirmed this participant is eligible for this programme. Your training provider will contact them directly."
 
       deferred_training:
-        id: "training_or_eligible_for_training"
-        label: "Training or eligible for training"
-        description: "We’ve confirmed these people are eligible for this programme. Your training provider will contact them directly."
+        id: "participant_deferred"
+        label: "Participant deferred"
+        description: "Your provider has reported that this particpant has deferred. Contact the provider if you think this is wrong."
```

Comment:
worth checking with Mark if the content on the design v2 is the latest or the document as they are slightly different